convert2rhel/actions/__init__.py

```diff
@@ -59,11 +60,12 @@
 #:      tests can be run.
 #:
 #: .. warning::
-#:      * Do not change the numeric value of these statuses once they
+#:      * Do not change the numeric value of these levels once they
 #:        have been in a public release as external tools may be depending on
 #:        the value.
-#:      * Actions should not set a status to ``SKIP``.  The code which
+#:      * Actions should not set a level to ``SKIP``.  The code which
 #:        runs the Actions will set this.
+# add info status code as 25
```

Comment:
```suggestion
```
It's probably better to track this in [RHELC-976](https://issues.redhat.com/browse/RHELC-976?focusedId=22507957&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-22507957)